packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -353,14 +353,14 @@ export function isNetwork(chainId: ChainId) {
 
```

Comment:
Removing additional networks displayed at the front end and only displaying nexus and holesky chain